java/integTest/src/test/java/glide/ErrorHandlingTests.java

```diff
@@ -12,10 +12,11 @@
 import glide.api.models.exceptions.RequestException;
 import java.net.ServerSocket;
 import java.util.concurrent.ExecutionException;
-import java.util.concurrent.TimeUnit;
 import lombok.SneakyThrows;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
 
+@Timeout(20)
```

Comment:
I kind of liked how the old solution told us what units are being used. I thought this was 20 minutes at first glance, until I saw the deleted code.

Reply:
I updated the PR description
`@Timeoout` is in seconds

Reply:
I meant more like if I were to just start reading these tests for the first time, I would not know what the units are. I think a comment might help with clarifying that here.

Reply:
Added